tools/assignmentPlacer.py

```diff
@@ -0,0 +1,28 @@
+import os, sys
+import shutil
+
+gradeable_path = '/var/local/submitty/courses/s20/sample/submissions/grades_released_homework_autohiddenEC'
+
+if __name__ == "__main__":
+    if not os.path.isdir('assignments') or not os.path.isdir(gradeable_path):
+        print('The assignments directory or the gradeable does not exist. Please create these.')
+        sys.exit(1)
+
+    files_in_assign = os.listdir('assignments')
+    num_to_place = min(len(os.listdir(gradeable_path)), len(files_in_assign))
+    placed = 0
+    ci = 0
+    gradeable_path_count = gradeable_path.count('/')
+    for root, dirs, files in os.walk(gradeable_path):
+        for name in dirs:
+            print(f'{root} {name}')
+            if (root.count('/') == gradeable_path_count):
```

Comment:
Trying to understand this check, is this to limit us to only searching the top-level of `gradeable_path` and no deeper? Are we actually walking through the entirety of `gradeable_path` and doing something meaningful with those layers?

Reply:
We are only walking into one child from the root (i.e. `gradeable_path/user_id` to find the highest submission number), I can change this to not use os.walk but rather a combinations of os.listdirs since the depth we use is fixed for each execution.